contents/ui-translations/ar.csv

```diff
@@ -211,6 +211,7 @@ Data imported,البيانات المستوردة
 Date added,تم إضافة التاريخ
 Date format,صيغة التاريخ
 Date modified,تاريخ التعديل
+Date Range,Date Range
```

Comment:
We do have key named `property daterange`. That key is used to translate the template property of date range type. Do you think that it would be worthwhile to reuse it?

Reply:
Using `property daterange` was my first approach but I thought that could be confusing, given your suggestion, I think it's fine to use the existing key and take advantage of already having some of the translations.